### PR TITLE
Chore: Use SelectAccountDropdownPo

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Be consistent. Use the SelectAccountDropdownPo instead of the DropdownPo directly.

# Changes

* Add methods to SelectAccountDropdownPo.
* TransactionFromAccountPo uses SelectAccountDropdownPo instead of DropdownPo directly. Change method names.
* Change the tests using TransactionFromAccountPo's dropdown.

# Tests

Only test changes.

# Todos

- [ ] Add entry to changelog (if necessary).

Not necessary.

### CODE REVIEW

**Comment by lmuntaner** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #4354** <a href="https://app.graphite.dev/github/pr/dfinity/nns-dapp/4354?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/dfinity/nns-dapp/4354?utm_source=stack-comment).

**Comment by dskloetd** [comment]
Can you add a PR description?

File: frontend/src/tests/lib/modals/neurons/NnsStakeNeuronModal.spec.ts [hunk 1]
```diff
@@ -127,7 +127,7 @@ describe("NnsStakeNeuronModal", () => {
         await po
           .getNnsStakeNeuronPo()
           .getTransactionFromAccountPo()
-          .getDropdownPo()
+          .getSelectAccountDropdownPo()
           .isPresent()
```
Comment (dskloetd) [inline frontend/src/tests/lib/modals/neurons/NnsStakeNeuronModal.spec.ts:131]: Should this be `.hasDropdown()`?

Comment (lmuntaner) [reply frontend/src/tests/lib/modals/neurons/NnsStakeNeuronModal.spec.ts:131]: Done, thanks!

**Comment by lmuntaner** [comment]
> Can you add a PR description?

Oops, sorry. Done!

**Comment by lmuntaner** [comment]
@dskloetd ready for another review

**Review by dskloetd — APPROVED** [review]

**Pull request merged**